public/src/components/epicTests/epicTestEditor.tsx

```diff
@@ -86,6 +87,7 @@ const styles = ({ spacing, typography}: Theme) => createStyles({
   },
   button: {
     marginTop: spacing(2),
+    marginLeft: spacing(2),
     float: 'right'
```

Comment:
Just wondering if float is redundant since the buttons are in a flexbox?

Reply:
good point, you're right!